src/Bicep.Core/CoreResources.resx

```diff
@@ -286,4 +286,11 @@
   <data name="MaxNumberResourcesRuleMessageFormat" xml:space="preserve">
     <value>Too many resources. Number of resources is limited to {0}.</value>
   </data>
+  <data name="UnusedResourceRuleDescription" xml:space="preserve">

```

Comment:
It would be good to be explicit that this only applies to 'existing' resources in the messages and possibly the name of the linter.